examples/research_projects/mulit_token_textual_inversion/README.md

```diff
@@ -0,0 +1,139 @@
+## This project
+
```

Comment:
The author of this project is [Isamu Isozaki](https://github.com/isamu-isozaki) - please make sure to tag the author for issue and PRs as well as @patrickvonplaten.